### DISCUSSION TITLE
Runtime Section: Launch Job from Cronjob

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [mia-platform/community#284] [Ideas]
### Which area of this project could be most improved?

Console

### Description

Since the console gives the opportunity to create cronjobs, it would be nice to launch them manually as [Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/job/) from the Runtime section.

This feature should replicate the behavior of the kubectl command:

```
kubectl create job --from=cronjob/<cronjob-name> <job-name>
```

This could be useful because sometimes we need to launch one-time execution of certain pods. Also, it would be nice to have a Job Section in the Desing, to write them more easily

**Comment by fredmaggiowski** [comment]
Hey, this is a nice feature to have, we had it in mind in the last few months but didn't find any real use-case scenario; if other people find it useful we might start doing it; can you please open a feature request in the internal board? @albertotessarotto FYI

**Reply by albertotessarotto** [reply]
Thanks for tracking this feature request in our internal backlog. Given the relevance of the reported use case, among the product evolutions we try to appropriately prioritize it.

**Comment by marcofilippi** [comment]
Hi! The feature of manually launching a CronJob from the Runtime Area has now been released with the Console v11.6.0!
